src/Capgemini.PowerApps.PackageDeployerTemplate/Services/TableColumnProcessingService.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace Capgemini.PowerApps.PackageDeployerTemplate.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using Capgemini.PowerApps.PackageDeployerTemplate.Adapters;
+    using Capgemini.PowerApps.PackageDeployerTemplate.Config;
+    using Microsoft.Crm.Sdk.Messages;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Xrm.Sdk;
+
+    /// <summary>
+    /// Service responsible for setting Auto-number seeds.
+    /// </summary>
+    public class TableColumnProcessingService
+    {
+        private readonly ILogger logger;
+        private readonly ICrmServiceAdapter crmSvc;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TableColumnProcessingService"/> class.
+        /// </summary>
+        /// <param name="logger">The <see cref="ILogger"/>.</param>
+        /// <param name="crmSvc">The <see cref="ICrmServiceAdapter"/>.</param>
+        public TableColumnProcessingService(ILogger logger, ICrmServiceAdapter crmSvc)
+        {
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            this.crmSvc = crmSvc ?? throw new ArgumentNullException(nameof(crmSvc));
+        }
+
+        /// <summary>
+        /// Processes tables with columns that have specific configurations, such as setting Auto-number seeds.
+        /// </summary>
+        /// <param name="tableConfigs">A collection of table elements to process specific attributes for.</param>
+        public void ProcessTables(IEnumerable<TableConfig> tableConfigs)
+        {
+            List<OrganizationRequest> requests = new List<OrganizationRequest>();
+
+            // Get all of the Auto-number seed configurations in the collection
+            requests.AddRange(this.GenerateAutonumberSeedRequests(tableConfigs));
+
+            // Check if we have requests that need processing.
+            if (requests.Count > 0)
+            {
+                this.logger.LogInformation("Executing requests for table columns.");
+                this.crmSvc.ExecuteMultiple(requests);
```

Comment:
Would be better to do something with the responses here. At the moment errors will be silent.

Reply:
There used to be (or maybe there still is) an extension method to `ILogger` to log `ExecuteMultiple` errors.

Reply:
There is! I'll add it!